seqr/views/apis/anvil_workspace_api_tests.py

```diff
@@ -239,12 +355,53 @@ def test_create_project_from_workspace(self, mock_tempfile, mock_mv_file, mock_f
                          .format(namespace=TEST_WORKSPACE_NAMESPACE, name=TEST_NO_PROJECT_WORKSPACE_NAME))
 
         # Test saving ID file exception
+        responses.calls.reset()
         url = reverse(create_project_from_workspace, args=[TEST_WORKSPACE_NAMESPACE, TEST_NO_PROJECT_WORKSPACE_NAME2])
         mock_mv_file.side_effect = Exception('Something wrong while moving the ID file.')
+        # Test triggering dag exception
+        responses.replace(responses.PATCH,
+                      '{}/api/v1/variables/AnVIL_WES'.format(MOCK_AIRFLOW_URL),
+                      json={'Error': '404 Client Error'},
+                      status=404)
+        
         response = self.client.post(url, content_type='application/json', data=json.dumps(REQUEST_BODY))
         self.assertEqual(response.status_code, 200)
-        mock_api_logger.error.assert_called_with('Uploading sample IDs to Google Storage failed. Errors: Something wrong while moving the ID file.',
-                      self.manager_user, detail=['HG00735', 'NA19675', 'NA19678'])
+        project2 = Project.objects.get(workspace_namespace=TEST_WORKSPACE_NAMESPACE, workspace_name=TEST_NO_PROJECT_WORKSPACE_NAME2)
+        
+        logger_calls = [
+            mock.call('Uploading sample IDs to Google Storage failed. Errors: Something wrong while moving the ID file.',
+                      self.manager_user, detail=['HG00735', 'NA19675', 'NA19678']),
+            mock.call(f'404 Client Error: Not Found for url: {MOCK_AIRFLOW_URL}/api/v1/variables/AnVIL_WES',
+                      self.manager_user)
+        ]
+        
+        mock_api_logger.error.assert_has_calls(logger_calls)
+        
+        slack_message_on_failure = """
+        ERROR triggering AnVIL loading for project {guid}: 404 Client Error: Not Found for url: {airflow_url}/api/v1/variables/AnVIL_WES 
+        
+        DAG seqr_vcf_to_es_AnVIL_WES_v0.0.1 should be triggered with following: 
+        ```{{
+    "active_projects": [
+        "{guid}"
+    ],
+    "vcf_path": "gs://test_bucket/test_path.vcf.gz",
+    "project_path": "gs://seqr-datasets/v02/GRCh38/AnVIL_WES/{guid}/v1",
+    "projects_to_run": [
+        "{guid}"
+    ]
+}}```
+        """.format(
+            guid=project2.guid,
+            airflow_url = MOCK_AIRFLOW_URL
+        )
+        mock_slack.assert_any_call(SEQR_SLACK_LOADING_NOTIFICATION_CHANNEL, slack_message_on_failure)
+        mock_send_email.assert_not_called()
+        self.assertEqual(len(responses.calls), 1)
+        self.assertEqual(responses.calls[0].request.url, '{}/api/v1/variables/AnVIL_WES'.format(MOCK_AIRFLOW_URL))
+        self.assertEquals(responses.calls[0].request.method, "PATCH")
+        self.assertEqual(responses.calls[0].request.headers['Authorization'], 'Bearer {}'.format(MOCK_TOKEN))
+        self.assertEquals(responses.calls[0].response.json(), {'Error': '404 Client Error'})
```

Comment:
this line isn't really necessary - since you configure the fake response in the test and we test for it in the failure message I think its better not to explicitly test the fake response here and only test the the request was sent as expected